demo/reactjs/README.md

```diff
@@ -2201,7 +2201,7 @@ or for a GitHub user page:
 or for a custom domain page:
 
 ```json
-  "homepage": "https://mywebsite.com",
+  "homepage": "http://mywebsite.com",
```

Comment:
This is a very odd special case. `mywebsite.com` should really be `example.<something>` (com, org, net, -- there are a number of choices) -- the domain `mywebsite.com` is for sale and could be squatted on at any moment by a malicious entity. There are 6 instances of `mywebsite.com` in this file, and only one of them uses `https`. I'm happy to change them all to use `https`, but I'd be happier to change all of them not to be `mywebsite.com`.